website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Announcing React Native 0.62 with Flipper
+author: Rick Hanlon
+authorTitle: React Native Core at Facebook
+authorURL: https://twitter.com/rickhanlonii
+authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
+authorTwitter: rickhanlonii
+category: announcements
+---
+
+Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
+
+## Flipper by default
+
+[Flipper](https://fbflipper.com/) is a developer tool for debugging mobile apps. It’s popular in the Android and iOS communities, and in this release we’ve enabled support by default for new and existing React Native apps.
+
+Flipper provides the following features out of the box:
+
+- Network Inspector
+- Logout Inspector
+- Crash Reporter
+- Device logs
+
+There is also a marketplace of custom Flipper plugins you can install:
+
+[TODO: Flipper Image]
+
+For more information, check out the Flipper documentation here.
+
+## Better dark mode support
+
+We’ve added a new Appearance module to provide access to the user's appearance preferences, such as their preferred color scheme (light or dark).
+
+```
+const colorScheme = Appearance.getColorScheme();
+if (colorScheme === 'dark') {
+  // Use dark color scheme
+}
+```
+
+We’ve also added a hook to subscribe to state updates to the users preferences automatically:
+
+```
+import {Text, useColorScheme} from 'react-native';
+
+const MyComponent = () => {
+  const colorScheme = useColorScheme();
+  return <Text>useColorScheme(): {colorScheme}</Text>;
+};
+```
+
+See the docs for [Appearance](https://reactnative.dev/docs/appearance) and [useColorScheme](https://reactnative.dev/docs/usecolorscheme) for more information.
+
+## Moving AppleTV to react-native-tvos
+
+As part of our Lean Core effort and to bring AppleTV in line with other platforms like React Native Windows and React Native MacOS, we’ve started to remove AppleTV specific code from core.
+
+Going forward, Apple TV support for React Native will be maintained in [react-native-community/react-native-tvos](https://github.com/react-native-community/react-native-tvos) along with the corresponding `react-native-tvos` NPM package. This is a full fork of the main repository, with only the changes needed to support Apple TV.
+
+Releases of `react-native-tvos` will be based on the public release of React Native. For this 0.62 release of `react-native` please upgrade AppleTV projects to use `react-native-tvos` 0.62.
+
+## Other Improvements
```

Comment:
as briefly mentioned in the Discord, I think that maybe it would be better to add a section (and I think the best spot would be here, before 'Other improvements') to talk about the new upgrading support repo. Something along the lines of what has been written here https://github.com/facebook/react-native-website/pull/1576

But reduced to one or 2 paragraphs and let the "heavy work" of explaining to the readme over there.

Something like:
```suggestion
## Even more upgrading support

When 0.61 was released, the community introduced a new tool called [upgrade-helper](https://react-native-community.github.io/upgrade-helper/) to better support developers in their upgrading effort. It has been a great addition and we are grateful for all the positive vibes that we felt around that project.

That said, given that upgrading can still be an hard task because of the inevitable evolution of the native platforms, we have decided to one-up the dedicated upgrading resources: we are eager to announce [Upgrade-Support](https://github.com/react-native-community/upgrade-support), a central place to request and give help when upgrading your app.

Upgrade Support is a GitHub repo hosting an issue tracker where people can post any issues they have while upgrading their projects. This should help increase the likelihood that people can find others with the same problems and work together to figure out a solution. For more information around this project, please refer to its [README](https://github.com/react-native-community/upgrade-support).

## Other Improvements
```